### ISSUE TITLE
Support incx and repeat parameters in add_region - FPGAGrid Rendering

### ISSUE THREAD

**Issue by ganeshgore** [ganeshgore/spydrnet-physical#17]

**Comment by ganeshgore** [comment]
@saadkhalil-rs can you confirm, if this still requires work, or if we already have this functionality

**Comment by saadkhalil-rs** [comment]
No, I rechecked incr and repeat with different scenarios both of them are working.  Was not able to incorporate a region by using both incr and repeat together. Actually VTR documentation also does not use them together, so do not need that functionality.

**Comment by saadkhalil-rs** [comment]
No, I rechecked incr and repeat with different scenarios both of them are working.  Was not able to incorporate a region by using both incr and repeat together. Actually VTR documentation also does not use them together, so do not need that functionality.
